dts/arm/st/h7/stm32h7.dtsi

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2019 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm/armv7-m.dtsi>
+#include <dt-bindings/clock/stm32_clock.h>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m7";
+			reg = <0>;
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			mpu: mpu@e000ed90 {
+				compatible = "arm,armv7m-mpu";
+				reg = <0xe000ed90 0x40>;
+				arm,num-mpu-regions = <8>;
```

Comment:
Any reason the MPU is in the CPU nodes?

Reply:
I actually followed the scheme provided for STM32F7.
It's a recent addtion from @ioannisg : https://github.com/zephyrproject-rtos/zephyr/pull/12173

Reply:
Is this Cortex-M3/4 @erwango ? then you don't really need that; it's going to be 8 for sure.
You need this for CM7 or CM33

Reply:
@ioannisg, this one is CM7.